core/src/main/java/hudson/model/Node.java

```diff
@@ -115,6 +118,18 @@ public boolean isHoldOffLaunchUntilSave() {
     }
 
     /**
+     * {@inheritDoc}
+     * @since 1.635
+     */
+    @Override
+    public void save() throws IOException {
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins != null) {
+            jenkins.updateNode(this);
+        }
```

Comment:
It would mean something was scheduled into a background thread that had the bad luck to run while Jenkins was shutting down, so the edit gets lost, oh well.